src/System.Runtime.Numerics/src/System/Numerics/BigInteger.cs

```diff
@@ -1174,6 +1174,17 @@ public byte[] ToByteArray(bool isUnsigned=false, bool isBigEndian=false)
         /// <returns>true if the bytes fit in <see cref="destination"/>; false if not all bytes could be written due to lack of space.</returns>
         /// <exception cref="OverflowException">If <paramref name="isUnsigned"/> is <c>true</c> and <see cref="Sign"/> is negative.</exception>
         public bool TryWriteBytes(Span<byte> destination, out int bytesWritten, bool isUnsigned=false, bool isBigEndian=false)
+        {
+            bytesWritten = 0;
+            if (TryGetBytes(GetBytesMode.Span, destination, isUnsigned, isBigEndian, ref bytesWritten) == null)
+            {
+                bytesWritten = 0;
+                return false;
+            }
+            return true;
+        }
+
+        public bool TryWriteOrCountBytes(Span<byte> destination, out int bytesWritten, bool isUnsigned = false, bool isBigEndian = false)
```

Comment:
Did you mean to introduce a new public member here? It isn't in the ref...

Reply:
Nope, meant for it to be internal. Will fix. Thanks.